kahu/deployment_guide.md

```diff
@@ -31,37 +31,71 @@ cd kahu/config/crd/v1beta1/bases/
 kubectl apply -f .
 ```
 
-4) Kahu provides nfs as the default way for backing up metadata. For using nfs as the metadata backup provider, nfs server need to be up and running.
+4) Kahu controller service watches on Kubernetes csi snapshot related CRD's. So these CRDs need to be installed as well.
+
+Based on the kubernetes version, deploy the respective version of CustomResourceDefinition(CRD).
+
+For version v1beta1:
+```shell
+cd ../../../../deploy/dependency/snapshotcrds/v1beta1
+kubectl apply -f .
+```
+
+For version v1:
+```shell
+cd ../../../../deploy/dependency/snapshotcrds/v1
+kubectl apply -f .
+```
+
+5) Kahu provides nfs as the default way for backing up metadata. For using nfs as the metadata backup provider, nfs server need to be up and running.
 This nfs server can be hosted anywhere and used provided it has connectivity and access with kahu provider.
 
-5) Alternatively we can also host an nfs server inside cluster with below steps. Create a nfs service and extract service ip address
+6) Alternatively we can also host an nfs server inside cluster with below steps. Create a nfs service and extract service ip address
 ```shell
 cd ../../../../deploy/yamls/nfsserver/
 kubectl apply -f nfs-server-service.yaml
 kubectl get service -n kahu | grep nfs-server
 ```
-6) Edit nfs-pv.yaml to update NFS-SERVER-IP-ADDR with service ip address obtained in step 5
+7) Edit nfs-pv.yaml to update NFS-SERVER-IP-ADDR with service ip address obtained in step 5
```

Comment:
as the step numbers changed  because of newly added steps..so please  change  from " service ip address obtained in step 5" to  "service ip address obtained in step 6"

Reply:
Updated